flexget/components/sites/sites/ettv.py

```diff
@@ -0,0 +1,44 @@
+from urllib.parse import urlparse
+
+from loguru import logger
+
+from flexget import plugin
+from flexget.components.sites.urlrewriting import UrlRewritingError
+from flexget.event import event
+from flexget.utils.soup import get_soup
+
+logger = logger.bind(name='ettv')
+
+DOMAINS = [
+    'www.ettv.to',
+    'www.ettvdl.com',
+]
+
+class UrlRewriteETTV:
```

Comment:
seems there's a missing line here. We style the code with black. You can install pre-commit and enable it for the repo and it will take care of this stuff

Reply:
fixed